src/EFCore.PG/Extensions/NpgsqlModelBuilderExtensions.cs

```diff
@@ -159,11 +219,64 @@ public static class NpgsqlModelBuilderExtensions
             return modelBuilder;
         }
 
+        /// <summary>
+        /// Registers a user-defined enum type in the model.
+        /// </summary>
+        /// <param name="modelBuilder">The model builder in which to create the enum type.</param>
+        /// <param name="name">The name of the enum type to create.</param>
+        /// <param name="labels">The enum label values.</param>
+        /// <returns>
+        /// The updated <see cref="ModelBuilder"/>.
+        /// </returns>
+        /// <remarks>
+        /// See: https://www.postgresql.org/docs/current/static/datatype-enum.html
+        /// </remarks>
+        /// <exception cref="ArgumentNullException">builder</exception>
+        [NotNull]
         public static ModelBuilder ForNpgsqlHasEnum(
             [NotNull] this ModelBuilder modelBuilder,
             [NotNull] string name,
             [NotNull] string[] labels)
-            => modelBuilder.ForNpgsqlHasEnum(null, name, labels);
+            => ForNpgsqlHasEnum(modelBuilder, null, name, labels);
```

Comment:
Why have you changed this?

Reply:
I _was_ thinking that this form delays a NRE when `modelBuilder` is `null` so that the other overload can throw from `Check.NotNull`... But on second thought, the above should be exactly the same in the IL (i.e. extension syntax is just syntactic sugar)...

I'll revert.